client/src/components/desk/CardList.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import {Container, Grid, Loader, Segment} from "semantic-ui-react";
+
+interface CardListProps {
+    title: string;
+    length: number;
+    loading?: boolean;
```

Comment:
seems reasonable but just checking, why is this optional? I might expect default loading state to be true

Reply:
So that if you don't explicitly include the `loading` attribute the code will still compile and `loading` is just false in that case